src/View/Components/FormComponent.php

```diff
@@ -6,7 +6,7 @@
 
 abstract class FormComponent extends Component
 {
-    public function render()
+    public function render(): mixed
```

Comment:
The correct type is `Closure`
```suggestion
    public function render(): Closure
```